src/main/java/org/stellar/sdk/Sep10Challenge.java

```diff
@@ -288,7 +296,7 @@ public static ChallengeTransaction readChallengeTransaction(String challengeXdr,
     return signersFound;
   }
 
-  private static Set<String> verifyTransactionSignatures(Transaction transaction, Set<String> signers) throws InvalidSep10ChallengeException {
+  private static Set<String> verifyTransactionSignatures(Transaction transaction, String domainName, Set<String> signers) throws InvalidSep10ChallengeException {
```

Comment:
it looks like `domainName` is not used anywhere in this function

Reply:
Sorry, fixed.